openfisca_web_site/templates/france/fr/tree/demo.mako

```diff
@@ -0,0 +1,234 @@
+                              id: "individu1",
+                              salaire_de_base: salaire2
+                            }
+                          ],
+                          menages: [
+                            {
+                              personne_de_reference: "individu0",
+                              conjoint:"individu1"
+                            }
+                          ]
+                        },
+                        period: "2016"
+                      }
+                    ],
+                    variables: ["revenu_disponible"],
+                    intermediate_variables: true
+          }
+        var scenarioString = JSON.stringify(scenarioObj)
+        return scenarioString
+        }
+      function appelALAPI(){
+        var xmlhttp = new XMLHttpRequest();
+        var url = "https://api.openfisca.fr/api/1/calculate";
+        var json = JSONDeLaSituation();
+        xmlhttp.open("POST", url, true);
+
+        xmlhttp.onreadystatechange=function() {
+          //console.log(xmlhttp.responseText)
+          if (xmlhttp.readyState==4 && xmlhttp.status==200) {
+            responseFromAPI = JSON.parse(xmlhttp.responseText);
+            //console.log(responseFromAPI || xmlhttp.responseText);
+            transformationDeLaDonnee(responseFromAPI);
+            }
+          };
+        xmlhttp.setRequestHeader("Content-Type", "application/json");
+        xmlhttp.send(json);
+        }
+      function transformationDeLaDonnee(responseFromAPI) {
+        var results = {};
+        results["revenu_disponible"] = Math.round(responseFromAPI.value[0].menages[0].revenu_disponible["2016"]);
+        results["impots_directs"] = -Math.round((responseFromAPI.value[0].menages[0].impots_directs["2016"]));
+        results["revenus_du_travail"] = 0;
+        for (var individu in responseFromAPI.value[0].individus){
+          results["revenus_du_travail"] += Math.round(responseFromAPI.value[0].individus[individu].revenus_du_travail["2016"]);
+            }
+        results["prestations_sociales"] = Math.round(responseFromAPI.value[0].familles[0].prestations_sociales["2016"]);
+        results["txDImposition"] = Math.round((results["impots_directs"])*100/(results["impots_directs"] + (results["revenu_disponible"])));
+        results["txDisponible"] = Math.round(100 - results["txDImposition"]);
+        results["salairesBruts"] = salaire1 + salaire2;
+        results["chargesSalariales"] = results["salairesBruts"] - results["revenus_du_travail"];
+        affichageDeLInformation(results);
+        };
+      function affichageDeLInformation(results){
+        document.getElementById("txtRevenuDisponible").innerHTML=`
+          <li>dispose d'un <strong>revenu disponible</strong> de ` + (Math.round(results["revenu_disponible"])).toLocaleString("fr") + ` €/an, c'est à dire ` + (Math.round(results["revenu_disponible"]/12)).toLocaleString("fr") + ` €/mois ;</li>
```

Comment:
This is quite hard to read. I'd suggest:
- Add this in the `HTML` document further down, with `{revenu_disponible}` as a placeholder.
- In this function, replace the placeholders by the actual result.

This is basically what templating engines usually do :).